common/skiplinks/reports/ally-1-fr.html

```diff
@@ -0,0 +1,15 @@
+---
+{
+	"title": "Assessment d'accessibilité #1 - Liens de saut de page",
+	"language": "fr",
+	"description": "Évaluation des critères de succès aux niveaux A et AA du WCAG 2.1",
+	"tag": "skiplinks",
+	"parentdir": "skiplinks",
+	"parentPage": "Liens de saut de page",
+	"parentPageURL": "skiplinks",
+	"altLangPage": "a11y-1-en.html",
+	"dateModified": "2023-07-06",
+	"layout": "assessment_wrote_en-fr",
+	"reportURL": "pre-a11y-1.json"
```

Comment:
```suggestion
	"reportURL": "pre-ally-1.json"
```

Reply:
switched all to use a11y for consistency